services/mlmodel/tflite_cpu/tflite_cpu.go

```diff
@@ -0,0 +1,262 @@
+// Package tflitecpu is meant to implement a tfliteCPU in the ML model service
+package tflitecpu
+
+import (
+	"context"
+	fp "path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+
+	"go.viam.com/rdk/config"
+	inf "go.viam.com/rdk/ml/inference"
+	"go.viam.com/rdk/ml/inference/tflite_metadata"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/services/vision"
+	"go.viam.com/rdk/utils"
+)
+
+func init() {
+	registry.RegisterService(vision.Subtype, resource.DefaultServiceModel, registry.Service{
+		Constructor: func(ctx context.Context, deps registry.Dependencies, config config.Service, logger golog.Logger) (interface{}, error) {
+			return newTFLiteCPUModel(ctx, deps, config, logger)
+		},
+	})
+	sModel := resource.NewDefaultModel("tflitecpu")
+	config.RegisterServiceAttributeMapConverter(mlmodel.Subtype, sModel, func(attributes config.AttributeMap) (interface{}, error) {
```

Comment:
resource.DefaultServiceModel will make the name of the model "builtin". Instead what you want to do is

```
var sModel = resource.NewDefaultModel("tflite_cpu")
func init() {
	registry.RegisterService(vision.Subtype, sModel, registry.Service{ ...}
        config.RegisterServiceAttributeMapConverter(mlmodel.Subtype, sModel, func(attributes config.AttributeMap) (interface{}, error) { ... }

```